projects/core/src/lib/users/oidc/oidc.component.ts

```diff
@@ -65,7 +66,10 @@ export class OidcComponent implements OnInit, OnDestroy {
                         this.user = session.user;
                     }
                 },
-                error: () => {},
+                error: () => {
+                    // guest login failed -> navigate to sign in
+                    this.router.navigate(['/signin']);
```

Comment:
Does this work in any case? Maybe route to `/` such that this works in any app?

Reply:
Routing it to `/` doesn't work. Nothing happens in the app. maybe because it already is at `/`?

Reply:
It seems like you can force a reload on the same location on extras/options:
https://angular.io/api/router/NavigationBehaviorOptions#onSameUrlNavigation